data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsSourceConfig.java

```diff
@@ -77,6 +76,10 @@ public class RdsSourceConfig {
     @JsonProperty("authentication")
     private AuthenticationConfig authenticationConfig;
 
+    @JsonProperty("tls")
+    @JsonAlias("ssl")
```

Comment:
Why do we need the alias?

Reply:
Just thought of allowing both to be used here.

Reply:
`ssl` is the old name. Since we have not released RdsSourceConfig. We should stick with `tls`.

Reply:
Makes sense. Removed `ssl`.